src/side-navigation/side-navigation.scss

```
@@ -99,18 +99,19 @@
       @include themed {
         border-color: t(iui-color-background-5);
       }
-    }
 
-    > .iui-button-icon {
-      width: $iui-icons-large;
-      height: $iui-icons-large;
-      flex-shrink: 0;
+      //boost specificity
+      > .iui-button-icon.iui-button-icon {
+        width: $iui-icons-large;
+        height: $iui-icons-large;
+        flex-shrink: 0;
 
-      + .iui-button-label {
-        margin-left: $iui-icons-large;
-        white-space: nowrap;
-        overflow: hidden;
-        text-overflow: ellipsis;
+        + .iui-button-label {
+          margin-left: $iui-icons-large;
+          white-space: nowrap;
+          overflow: hidden;
+          text-overflow: ellipsis;
+        }
```

Will gap not work here? I see you unset it at the top, but you could just change it to `$iui-icons-large`.

If not, we could still remove the dependency on iui-button-label
```suggestion
        &:not(:last-child) {
          margin-right: $iui-icons-large;
        }
```

And those overflow rules don't need to be nested this deep so you could add those somewhere above.

---

I see you went with the suggested change. Did `gap: $iui-icons-large` not work?

---

`gap` did not work.

---

Are you sure? It seems to work on my end. All I did is remove `margin-right: $iui-icons-large` and set `gap: $iui-icons-large` on the parent.

---

Fixed it. Your suggestion worked.